integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/iceberg/SparkIcebergCatalogIT.java

```diff
@@ -386,6 +421,88 @@ private void testDeleteMetadataColumn() {
     Assertions.assertEquals(0, queryResult1.size());
   }
 
+  private void testIcebergDeleteOperation() {
+    getIcebergTablePropertyValues()
```

Comment:
Could you use `ParameterizedTest ` to auto-generate parameters?

Reply:
got it.

Reply:
> Could you use `ParameterizedTest ` to auto-generate parameters?

@FANNG1 how about using ` @MethodSource("getIcebergTablePropertyValues")` ?

Reply:
ok

Reply:
> ok

got it.

Reply:
done